src/lib.rs

```diff
@@ -39,12 +39,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#![feature(asm)]
+#[feature(asm)]
```

Comment:
Why this change? I'm not sure what an outer attribute does if it's not actually attached to anything, but I think this should really decorate the module. It may not have mattered in your case, depending on the toolchain you used to build it. If `asm` has been stabilized, then it would work the same either way. But in `omicron` for example, we use an older toolchain that would cause this to break, I believe.